tests/ci/style_check.py

```diff
@@ -126,7 +126,7 @@ def commit_push_staged(pr_info: PRInfo) -> None:
     if not git_staged:
         return
     remote_url = pr_info.event["pull_request"]["base"]["repo"]["ssh_url"]
-    git_runner(f"{GIT_PREFIX} commit -m 'Automatic style fix'")
+    git_runner(f"{GIT_PREFIX} commit -m 'Automatic style fix' --fixup HEAD")
```

Comment:
I never seen the fixup argument being used together with commit message. Does it work like that? What's in the commit message then?

Reply:
it creates commit with next content
```
fixup! <comment of referenced commit>
<comment for this commit>
```
later, on `git rebase -i <commit>` all fixup commits will be tagget as fixup

Reply:
Ok, never tried it..

But still, we can't spoil the user's branch. And we shouldn't force him to do `rebase`.

It's a good intention but will meet strong resistance.